akka-remote/src/main/resources/reference.conf

```diff
@@ -659,13 +659,15 @@ akka {
         # Protocol to use for SSL encryption, choose from:
         # TLS 1.2 is available since JDK7, and default since JDK8:
         # https://blogs.oracle.com/java-platform-group/entry/java_8_will_use_tls
+        # TLS 1.3 is available since JDK 11:
+        # https://www.oracle.com/technetwork/java/javase/11-relnote-issues-5012449.html
```

Comment:
do we dare to say that we support that without any testing?

Reply:
Defintely not (especially considering https://github.com/akka/akka/issues/28092#issuecomment-547743545).

Reply:
(slightly off-topic) TIL, ["TLS 1.3 is not directly compatible with previous versions"](https://www.oracle.com/technetwork/java/javase/11-relnote-issues-5012449.html) so if/when we want to support it we may need to find a way to support rolling upgrades.